bgpd/bgp_evpn.c

```diff
@@ -4333,6 +4333,57 @@ static void update_autort_vni(struct hash_bucket *bucket, struct bgp *bgp)
 	}
 }
 
+/*
+ * Handle autort change for L3VNI.
+ */
+static void update_autort_l3vni(struct bgp *bgp)
```

Comment:
I'm just thinking, can't this be handled inside update_autort_vni() without creating an additional function? Seems there are already common calling functions with the same `if` branches almost.